deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -147,6 +147,11 @@ spec:
               required:
               - gitSource
               type: object
+            enableIstio:
+              description: EnableIstio annotates the pods managed by the operator
```

Comment:
maybe do not repeat the property name on the description?

Reply:
Golint, but I guess we can remove it. I'll try. This is generated code from the source.